src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java

```diff
@@ -96,6 +99,16 @@ public AppiumServiceBuilder() {
         withEnvironment(System.getenv());
     }
 
+    @Override public int score(Capabilities capabilites) {
```

Comment:
What is the purpose of this method?

Reply:
Grid sessions are going to be created only if the capability score is greater than zero. Higher the score higher the probability of getting sessions created sooner as per my chat with Simon. so we need this, probably used in future versions of grid.

Reply:
It might be useful to add a docstring there with an explanation